perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/cpu_jpegdecoder.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 AutoCore, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef AUTOWARE__TENSORRT_BEVDET__CPU_JPEGDECODER_HPP_
+#define AUTOWARE__TENSORRT_BEVDET__CPU_JPEGDECODER_HPP_
+
+#include "common.hpp"
+
+#include <vector>
+
+int decode_cpu(
```

Comment:
The `decode_cpu` function uses `int` for the return type.  Considering that the function handles image data, which can potentially involve large numbers (e.g.,  representing image dimensions or pixel data size exceeding the capacity of a 32-bit integer), it's safer to use `int64_t` for the return type to avoid potential integer overflow issues.  This ensures compatibility across different architectures and prevents unexpected behavior. Please consider changing the return type to `int64_t`.